packages/caver-transaction/src/utils/accessTuple.js

```diff
@@ -0,0 +1,127 @@
+/*
+    Copyright 2022 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const _ = require('lodash')
+const utils = require('../../../caver-utils/src')
+
+/**
+ * An object defines the access tuple.
+ *
+ * @typedef {object} module:Transaction.AccessTupleObject
+ * @property {string} address - The address string.
+ * @property {Array.<string>} storageKeys - An array of the storage keys.
+ */
+/**
+ * Represents an access tuple that has an address and storage keys.
+ * AccessTuple is the element type of the access list.
+ * @class
+ */
+class AccessTuple {
+    /**
+     * Creates an access tuple.
+     *
+     * @example
+     * const result = caver.transaction.utils.accessTuple.create('0x{address in hex}', [ '0x{storage key}' ])
+     *
+     * @param {string} address - The address string.
+     * @param {Array.<string>} storageKeys - An array of the storage keys.
+     * @return {AccessTuple} An access tuple.
+     */
+    static create(address, storageKeys) {
+        return new AccessTuple(address, storageKeys)
+    }
+
+    /**
+     * Creates an access tuple.
+     * @constructor
+     * @param {string} address - The address string.
+     * @param {Array.<string>} storageKeys - An array of the storage keys.
+     */
+    constructor(address, storageKeys) {
+        this.address = address
+        this.storageKeys = storageKeys
+    }
+
+    /**
+     * @type {string}
+     */
+    get address() {
+        return this._address
+    }
+
+    set address(addr) {
+        if (!_.isString(addr)) throw new Error(`Invalid address type: Address should be string type ${typeof addr}`)
+        if (!utils.isAddress(addr)) throw new Error(`Invalid address: ${addr}`)
+        addr = utils.addHexPrefix(addr)
+        this._address = addr
+    }
+
+    /**
+     * @type {Array.<string>}
+     */
+    get storageKeys() {
+        return this._storageKeys
+    }
+
+    set storageKeys(keys) {
+        if (!_.isArray(keys)) keys = [keys]
+        for (let k of keys) {
+            if (!_.isString(k)) throw new Error(`Invalid storageKey type: Storage key should be string type ${typeof k}`)
+            if (!utils.isHex(k)) throw new Error(`Invalid storageKey: Storage key should be a hex string ${k}`)
```

Comment:
We should check whether length of storage key(with hex prefix) is `66` or not.
It because the storage key must be 32 bytes always by its spec.

Reply:
I added in https://github.com/klaytn/caver-js/pull/600/commits/208c01a1cbbe0191ef47577909eedac9a59022b9. Please check !